resources/views/livewire/slack-settings-form.blade.php

```diff
@@ -0,0 +1,104 @@
+                @else
+                    <input type="text" wire:model.lazy="slack_endpoint" id="slack_endpoint" class= 'form-control' placeholder="'https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX'" {{old('slack_endpoint', $slack_endpoint)}} ><br>
+{{--                                    {{ Form::text('slack_endpoint', old('slack_endpoint', $setting->slack_endpoint), array('class' => 'form-control','placeholder' => 'https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX', 'id' => 'slack_endpoint')) }}--}}
+                @endif
+                {!! $errors->first('slack_endpoint', '<span class="alert-msg" aria-hidden="true">:message</span>') !!}
+            </div>
+        </div>
+
+        <!-- slack channel -->
+        <div class="form-group required {{ $errors->has('slack_channel') ? 'error' : '' }}">
+            <div class="col-md-2">
+                {{ Form::label('slack_channel', trans('admin/settings/general.slack_channel')) }}
+            </div>
+            <div class="col-md-10">
+                @if (config('app.lock_passwords')===true)
+                    <input type="text" wire:model.lazy="slack_channel" id="slack_channel" class='form-control' placeholder="#IT-Ops" value="{{old('slack_channel', $slack_channel)}}" ><br>
+                    <p class="text-warning"><i class="fas fa-lock"></i> {{ trans('general.feature_disabled') }}</p>
+
+                @else
+                    <input type="text" wire:model.lazy="slack_channel" id="slack_channel" class= 'form-control' placeholder="#IT-Ops" value="{{old('slack_channel', $slack_channel)}}" ><br>
+                @endif
+                {!! $errors->first('slack_channel', '<span class="alert-msg" aria-hidden="true">:message</span>') !!}
+            </div>
+        </div>
+
+        <!-- slack botname -->
+        <div class="form-group required {{ $errors->has('slack_botname') ? 'error' : '' }}">
+            <div class="col-md-2">
+                {{ Form::label('slack_botname', trans('admin/settings/general.slack_botname')) }}
+            </div>
+            <div class="col-md-10">
+                @if (config('app.lock_passwords')===true)
+{{--                    {{ Form::text('slack_botname', old('slack_botname', $setting->slack_botname), array('class' => 'form-control','disabled'=>'disabled','placeholder' => 'Snipe-Bot')) }}--}}
+                    <input type="text" wire:model.lazy="slack_botname" id="slack_botname" class= 'form-control' placeholder="'Snipe-Bot'" {{old('slack_botname', $slack_botname)}} ><br>
+                    <p class="text-warning"><i class="fas fa-lock"></i> {{ trans('general.feature_disabled') }}</p>
+
+                @else
+                    <input type="text" wire:model.lazy="slack_botname" id="slack_botname" class= 'form-control' placeholder="'Snipe-Bot'" {{old('slack_botname', $slack_botname)}} ><br>
```

Comment:
Same placeholder comment as above.

```suggestion
                    <input type="text" wire:model.lazy="slack_botname" id="slack_botname" class= 'form-control' placeholder="Snipe-Bot" {{old('slack_botname', $slack_botname)}} ><br>
                    <p class="text-warning"><i class="fas fa-lock"></i> {{ trans('general.feature_disabled') }}</p>

                @else
                    <input type="text" wire:model.lazy="slack_botname" id="slack_botname" class= 'form-control' placeholder="Snipe-Bot" {{old('slack_botname', $slack_botname)}} ><br>
```